andela/ah-shakas-frontend
# #160577449 implement social sharing

## What does this PR do?
- Implements sharing of articles across various social platforms
## Description of Task to be completed?
- A user should be able to share/post/tweet articles on social media platforms like Facebook and Twitter.
## How should this be manually tested?
- Clone this branch ft-bookmark-160577455
- Run the following commands on your terminal:
   ` yarn install` - to install required dependencies
    `yarn start` - to start the server
- When the server starts, you will be redirected to the browser, where you can view articles.
- Notice the social medial icons to the far right, click on any of them to share an article.

**Screenshot**
![image](https://user-images.githubusercontent.com/34505753/49141666-2ae8fc00-f308-11e8-82b6-f2da141c9c23.png)

## Any background context you want to provide?
N/A
## What are the relevant pivotal tracker stories?
[160577449](https://www.pivotaltracker.com/n/projects/2198533/stories/160577449)

## Deployment note
N/A

## Related PRs branch
[ft-social-sharing-160577449)](https://github.com/andela/ah-shakas-frontend/tree/ft-social-sharing-160577449)

## Todos
